napari/utils/interactions.py

```diff
@@ -255,7 +256,7 @@ def __init__(self, shortcut: str):
         self._values = shortcut.split('-')
         for v in self._values:
             if len(v) > 1:
-                assert v in KEY_SYMBOLS.keys()
+                assert v in KEY_SYMBOLS.keys(), v
```

Comment:
Could we use try/except instead of `assert` and raise an exception/warning?

Reply:
agreed, even though this wasn't an issue you introduced, it would be great if you could clean it up while you're here!